### DISCUSSION TITLE
Would it be ok for us to add to homebrew (mac)?

### DISCUSSION THREAD

**Discussion by ImpossibleWorks** [cpvalente/ontime#643] [Ideas]
Would it be ok for us to add to homebrew? It might not be accepted as it's an unsigned binary, but worth a try. Thoughts?

**Comment by cpvalente** [comment]
Hi @ImpossibleWorks , this is a great idea.

I dont have objections with it, but as you mention it might not be accepted. From the brew docs on reject reasons

> App fails with GateKeeper enabled on Homebrew supported macOS versions and platforms (e.g. unsigned apps fail on Macs with Apple silicon/ARM).

However this makes me believe we might be able to use the old intel version which doesnt fail.

Is this something that we need to do or can the community (you for example) get started?
We have a few things moving at the moment and might take some time to get around to do it, but I am happy to help if someone is taking a lead

**Comment by ImpossibleWorks** [comment]
Happy to take the lead on this.

Is there a future where the arm64 version can be signed?

**Reply by cpvalente** [reply]
Nice! That is great to hear @ImpossibleWorks 

As for your question:
I find the documentation about this unclear.
It seems to hint at the fact that signing should be possible with a free account but I havent found any resources on how to achieve it.

In short: unless there is some sponsorship for the certificate, or a developer can help us figure it out, it is unlikely that this would ever be signed

**Reply by cpvalente** [reply]
Hi @ImpossibleWorks , 

With v3 we have opened a sponsorship programme for Ontime.
I have frontend the first year of subscriptions and the distribution of Ontime v3 for MacOS is now signed.

Hope this is good news and we can move forwards with this.
Let me know if I can help
